MapboxCoreNavigation/FixLocation.swift

```diff
@@ -4,12 +4,25 @@ import MapboxNavigationNative
 
 extension FixLocation {
     convenience init(_ location: CLLocation) {
+        var speedAccuracy: NSNumber? = nil
+        if #available(iOS 10.0, *) {
```

Comment:
These check seems redundant. `MapboxCoreNavigation` minimum iOS version is 10.0

Reply:
Thanks, check was removed.